Dockerfile-test

```diff
@@ -32,11 +48,21 @@ COPY --from=builder $SRC_PATH/docker/random-stopper.sh /usr/local/bin/random-sto
 COPY --from=builder $SRC_PATH/docker/random-killer.sh /usr/local/bin/random-killer.sh
 COPY --from=builder $SRC_PATH/docker/wait-killer-stopper.sh /usr/local/bin/wait-killer-stopper.sh
 COPY --from=builder $SRC_PATH/docker/cluster-restart.sh /usr/local/bin/cluster-restart.sh
+COPY --from=builder $GOPATH/bin/ipfs /usr/local/bin/ipfs
+COPY --from=builder /tmp/su-exec/su-exec /sbin/su-exec
+COPY --from=builder /tmp/tini /sbin/tini
+COPY --from=builder /etc/ssl/certs /etc/ssl/certs
 
-RUN mkdir -p $IPFS_CLUSTER_PATH && \
+RUN apk update && apk add bash && \
+    mkdir -p $IPFS_CLUSTER_PATH && \
+    adduser -D -h $IPFS_PATH -u 1000 -G 100 ipfs && \
+    chown 1000:100 $IPFS_PATH && \
     chown 1000:100 $IPFS_CLUSTER_PATH
 
+VOLUME $IPFS_PATH
 VOLUME $IPFS_CLUSTER_PATH
-ENTRYPOINT ["/sbin/tini", "--", "/usr/local/bin/start-daemons.sh"]
 
+ENV IPFS_LOGGING ""
+
+ENTRYPOINT ["/sbin/tini", "--", "/usr/local/bin/start-daemons.sh"]
```

Comment:
I didn't catch this earlier, but if we use tini as the entrypoint I believe it is going to intercept signals sent to the testing containers and break existing tests.  To be fair we should be reaping zombie processes in the test container, but I'll need to think about how to achieve this without blocking signals.